### PR TITLE
Fix app bar colors when empty task fragment is shown in two pane mode…

### PR DESCRIPTION
**Pull request by lemonboston**
…. #353

---

I did it the other way around, so set the list app bar color to the primary color when the empty fragment is shown, since no task(list) is selected that time. This is easier to implement and also safer I think, instead of somehow keeping the color of the last selected task.

### CODE REVIEW

**Comment by lemonboston** [comment]
I guess it's better to use primary color from the theme attribute.... I am going to update it.

**Comment by lemonboston** [comment]
Imported `bolts-color` and use `AttributeColor(R.attr.colorPrimary)` now for the app bar color.
Ready for check.

File: opentasks/src/main/java/org/dmfs/tasks/ViewTaskFragment.java [hunk 1]
```diff
@@ -175,23 +176,23 @@ public void run()
...
+        void onDelete(Uri taskUri);
 
         /**
          * Notifies the listener about the list color of the current task.
          *
          * @param color
          *         The color.
          */
-        public void updateColor(int color);
+        void updateColor(@ColorInt int color);
```
Comment (dmfs) [inline opentasks/src/main/java/org/dmfs/tasks/ViewTaskFragment.java]: How about changing this to take a `Color` instead?

Comment (lemonboston) [reply opentasks/src/main/java/org/dmfs/tasks/ViewTaskFragment.java]: I've changed it to use `Color`, also created https://github.com/dmfs/opentasks/issues/522 to use `Toned` as next step.
Ready for check.

**Pull request merged**